core-plugins/docs/RowDenormalizer-batchaggregator.md

```diff
@@ -10,39 +10,60 @@ Use Case
 The transform takes input record that stores a variable set of custom attributes for an entity, denormalizes it on the basis of the key field, and then returns a denormalized table according to the output schema specified by the user.
 The denormalized data is easier to query.
 
+Conditions
+----------
```

Comment:
would be better to move this below the properties section, since you have to understand the properties to understand what this section talks about